AppCenterPush/AppCenterPush/Internal/MSPushPrivate.h

```diff
@@ -17,14 +18,14 @@ static NSString *const kMSPushNotificationOldCustomDataKey = @"mobile_center";
 @protocol MSCustomApplicationDelegate;
 
 #if TARGET_OS_OSX
-@interface MSPush () <NSUserNotificationCenterDelegate>
+@interface MSPush () <NSUserNotificationCenterDelegate, MSAuthTokenContextDelegate>
 #else
-@interface MSPush ()
+@interface MSPush () <MSAuthTokenContextDelegate>
 #endif
 
 @property(nonatomic) id<MSPushDelegate> delegate;
 
-@property(nonatomic) NSString *pushToken;
+@property(atomic) NSString *pushToken;
```

Comment:
Then according to https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1309/files#r258166953 this one should be copy too.

Reply:
Good catch, thanks a lot.

Reply:
yes, it should! Sorry for missing this!